$MirOS: ports/multimedia/livestream/patches/patch-testProgs_playCommon_cpp,v 1.1.7.1 2005/03/18 15:50:47 tg Exp $
--- testProgs/playCommon.cpp.orig	Thu May 12 01:40:47 2005
+++ testProgs/playCommon.cpp	Thu May 19 19:58:58 2005
@@ -1253,7 +1253,7 @@ void checkForPacketArrival(void* /*clien
     struct timeval timeNow;
     gettimeofday(&timeNow, NULL);
 	char timestampStr[100];
-	sprintf(timestampStr, "%ld%03ld", timeNow.tv_sec, timeNow.tv_usec/1000);
+	sprintf(timestampStr, "%lld%03ld", (long long)timeNow.tv_sec, timeNow.tv_usec/1000);
     *env << (syncStreams ? "Synchronized d" : "D")
 		<< "ata packets have begun arriving [" << timestampStr << "]\007\n";
     return;
